README.md

```diff
@@ -72,4 +72,111 @@ to GHCR. In order to accomplish this task:
 2. Build the new image with the specific version you want: `docker build -t ghcr.io/sage-bionetworks-workflows/orca-recipes:0.0.0 .`
 3. Ensure you are logged into the GHCR: `echo $GITHUB_TOKEN | docker login ghcr.io -u USERNAME --password-stdin`
    1. Replace `$GITHUB_TOKEN` and `USERNAME` with the correct values
-4. Push the image to GHCR for your version: `docker push ghcr.io/sage-bionetworks-workflows/orca-recipes:0.0.0`
\ No newline at end of file
+4. Push the image to GHCR for your version: `docker push ghcr.io/sage-bionetworks-workflows/orca-recipes:0.0.0`
+
+
+## Contributing a Challenge DAG
+
+This section describes how you can contribute a new challenge DAG by configuring the `challenge_configs.yaml` file
+and the DAG factory pattern in this repository.
+
+----
+
+### TL;DR
+
+To add a DAG for your challenge, make a pull request introducing a profile to the `challenge_configs.yaml` file with the parameters listed in [Creating Your Challenge DAG Config](#creating-your-challenge-dag-config).
+See below for a template you can follow.
+
+Modify everything in `<>` and remove the `<>` when complete:
+
+```
+<my-challenge>:
+  synapse_conn_id: "<MY_SYNAPSE_CONN_ID>"
+  aws_conn_id: "<MY_AWS_CONN_ID>"
+  revision: "<1a2b3c4d>"
+  challenge_profile: "<my_challenge_profile>"
+  tower_conn_id: "<MY_CHALLENGE_PROJECT_TOWER_CONN>"
+  tower_view_id: "<syn123>"
+  tower_compute_env_type: "spot"
+  bucket_name: "<my-challenge-project-tower-scratch>"
+  key: "<10days/my_challenge>"
+  dag_config:
+    schedule_interval: "*/1 * * * *"
+    start_date: "2024-04-09T00:00:00"
+    catchup: false
+    default_args:
+      retries: 2
+    tags:
+      - "nextflow_tower"
+```
+
+Note that while `synapse_conn_id` and `aws_conn_id` are customizable, we do have connection IDs already in-place
+for you to use that will connect you to Synapse (through DPE's ORCA Service Account) and to AWS. But feel free to swap
+these for your own if they do not suit your needs.
+
+```
+synapse_conn_id: "SYNAPSE_ORCA_SERVICE_ACCOUNT_CONN"
+aws_conn_id: "AWS_TOWER_PROD_S3_CONN"
+```
+
+----
+
+### Overview
+
+As part of propping up a new Synapse-hosted challenge, the workflow you choose in [nf-synapse-challenge](https://github.com/Sage-Bionetworks-Workflows/nf-synapse-challenge/tree/main) will need to
+be automated to run for challenge submissions on a rolling basis. For this we leverage Airflow DAGs to perform these
+automated executions of your customized challenge workflow.
```

Comment:
Should this portion that explains the purpose of the challenge dag be moved to the top of README under `## Contributing a Challenge DAG`? The purpose is that as one is reading through the README they'll get context for what they're about to read first.